sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -216,7 +216,7 @@ messageBatch.TryAdd(new ServiceBusMessage(Encoding.UTF8.GetBytes("First")));
 messageBatch.TryAdd(new ServiceBusMessage(Encoding.UTF8.GetBytes("Second")));
 
 // send the message batch
-await sender.SendBatchAsync(messageBatch);
+await sender.SendAsync(messageBatch);
```

Comment:
Can we update SendBatchAsync -> SendAsync in the [Sending messages section](https://github.com/Azure/azure-sdk-for-net/blob/b274f04a3179abc53624e85dfa943f6b0080f6e0/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md#sending-messages)  and in the description of[ Sending and receiving batch of messages](https://github.com/Azure/azure-sdk-for-net/blob/b274f04a3179abc53624e85dfa943f6b0080f6e0/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md#sending-and-receiving-a-batch-of-messages)